tests/test_model.py

```diff
@@ -331,19 +331,19 @@ def check_linker(actual, expected_id, expected_sequence):
 
     for args in ((LinkerComponent(**linkers[0]), 'sequence:ACGT', 'ACGT'),
                  (LinkerComponent(**linkers[1]), 'sequence:T', 'T'),
-                 (LinkerComponent(**linkers[2]), 'sequence:AAAAA', 'AAAAA')):
+                 (LinkerComponent(**linkers[2]), 'sequence:actgu', 'ACTGU')):
         check_linker(*args)
 
     # check base validation
     with pytest.raises(ValidationError) as exc_info:
         LinkerComponent(**{
             'linker_sequence':
                 {
-                    'id': 'sequence:ABGT',
-                    'sequence': 'ABGT'
+                    'id': 'sequence:ACT1',
+                    'sequence': 'ACT1'
                 }
         })
-    msg = 'Linker sequence must consist only of {A,C,G,T}'
+    msg = 'string does not match regex "^[A-Z*\\-]*$"'
```

Comment:
Shouldn't it be "^[a-zA-Z*\\-]*$" ?

Reply:
This is the regex VRS uses: https://github.com/ga4gh/vrs/blob/abd88cd5a99cef598de6f0e4b70b028ebbde2888/schema/vrs.yaml#L618

Reply:
I’m not really sure how to change the error message.

Reply:
Updated. I am going to make an issue to have clearer error msgs rather than using "string does not.."